Developed

Pcontrerasm opened the pull request.

naxinya committed:
bootstrap

naxinya committed:
,

Agorr committed:
avance index

DiegoZapataT committed:
Tutoriales + Ruteo y otras cosas

Se realiza avance de pestaña tutoriales (aun faltan cosas) y su ruteo. 
se modifican archivos ya subidos por compañeros.

naxinya committed:
agregue carpetas css y otras al proyecto

naxinya committed:
.

naxinya committed:
.

DiegoZapataT committed:
Css + Bootstrap

Se logra ver el diseño de la pagina si se accede a ella desde el explorador de archivos, pero al ejecutar desde el servidor aun no se muestra el diseño, se arreglara pronto

DiegoZapataT committed:
Se crean vistas y rutas

Se crean vistas y rutas para moda, regresion lineal y promedio, se espera la creacion de los graficos para cada pestaña

Agorr committed:
bootstrap funcionando

funciona en index de carpeta templates/Integracion

naxinya committed:
bootstrap completo + pagina faq

bootstrap en todas las paginas de templates/integracion y creacion de pagina faq

Agorr committed:
conexion db

Conexión con base de datos mongodb alojada en la nube. Es necesario instalar djongo para que funcione. Usar comando "pip install djongo" en carpeta de proyecto.

Agorr committed:
buscar data db

naxinya committed:
grafico implementado

en moda.html

Agorr committed:
avance

Agorr committed:
grafico funcional con base de datos mongodb

Pcontrerasm committed:
Add files via upload

naxinya committed:
completando página faq

naxinya committed:
mas graficos 

graficos en paginas de regresion lineal y promedio

shofito committed:
fix en gráficos

función clear() para liberar memoria de los gráficos después de usarlos, ya no es necesario reiniciar la página al cambiar de gráfico

Commit:
generador de gráficos y formularios sobre una pagina ficticia para probarlos en un entorno simplificado antes de agregar al real.

shofito committed:
Eliminar código repetido

Las plantillas de los gráficos heredan de tipo_grafico.html

DiegoZapataT committed:
Pagina para grafico

Se crea ruteo y pagina para comenzar el trabajo del grafico correlacional

shofito committed:
fix direcciones

arreglo de direcciones antiguas y unas funciones borradas

DiegoZapataT committed:
Paginas tutoriales 

Se agregan las paginas con texto para aprender a programar en python, se insertaran videos en un futuro

naxinya committed:
nuevo index / elegir grafico

naxinya committed:
arreglo detalle footer

shofito committed:
Revert "gráfico de correlación"

This reverts commit 9ecd58b68aca862e2f9a622744ec8f4cfe229d14.

shofito committed:
gráfico de correlación

usa librerias las pandas y seaborn

shofito committed:
gráfico de correlación

usa las librerias pandas y seaborn

naxinya committed:
subir archivos json y ver algunosdatos en promedio

Agorr committed:
sacar datos

shofito committed:
pag para ver lista de colecciones (bd's)

usa pymongo, falta filtro

shofito committed:
obtener lista de colecciones y datos 

falta obtener la coleccion seleccionada, se muestra una (historiales)

jcredberry commented on Integracion/mongoconnect/views.py:
```diff
@@ -2,6 +2,7 @@
 from django.views.decorators.csrf import csrf_exempt
+from bson.json_util import dumps
 import pymongo
 # Create your views here.
```
Recuerden que si usan código de terceros, deben refenciar su origen y la licencia que con el que se comparte.

DiegoZapataT committed:
Avance de Paulina

Subo yo el avance de paulina ya que ella tiene problemas con GIT, por lo cual yo como lider de grupo lo pase en limpio y lo subo.

naxinya committed:
ver datos cualquier json masomenos

Agorr committed:
funcion recursiva

DiegoZapataT committed:
Se edita frontend

Se eliminan pestañas de graficos dejando solo 2 para su posterior implementacion en cada una, se modifica la side bar, modificacion del menu del frontend en general

DiegoZapataT committed:
Creacion formulario para Consultas 

Formulario de consultas para graficar, aun sin implementar la base de datos

DiegoZapataT committed:
Consultas y Graficos

Se Crea las consultas para graficar usando javascript, de manera que se muestren graficos desde nombre paciente hasta centro de salud.

DiegoZapataT committed:
Borre Basura

Borre Basura de los codigos que envie denante

shofito committed:
fix detalle en pag para ver datos

shofito committed:
pag para ver datos muestra lista de resultados por ..páginas

necesita un diseño mejor, se perdió el json "bonito"

DiegoZapataT committed:
Variables no usadas eliminadas

DiegoZapataT committed:
Se añade mas opciones para consultas estaticas

Se grafica fecha de nacimiento del paciente y su lugar de residencia

shofito committed:
otra version del grafico (de moda)

"grafico version 2" (ruta /graficador) tiene consultas reutilizables¿?, sirven para toda coleccion

Agorr committed:
formulario consultas avanzadas

shofito committed:
Fix conexión a base de datos

La bd se llama historiales y la colección se llama arquetipos, si esta bd no está en el servidor local (en sus pc ekisdé) hay que ejecutar la página de gráfico (de moda, el original no el versión 2) y consultar por algún dato  y listo

shofito committed:
fix en página para ver datos

 ahora muestra bien el json, click al id

DiegoZapataT committed:
FixGrafico principal scatter

Fixeo formulario para graficos con scatter

shofito committed:
página para ver datos de frecuencia de un paciente dado su rut

 la ruta es /moda/<str:rut>/, se cambió la ruta del gráfico a trabajar a /moda, en la página del gráfico a trabajar se encuentra el formulario para buscar por rut, en la página para obtener info del paciente falta arreglar el diseño de la lista de sesiones

shofito committed:
fix en los gráficos

arreglos en el diseño y en los datos que recibe la función plotly.newplot

naxinya committed:
.

shofito committed:
fixdiseño graficos

DiegoZapataT committed:
Avance de paulina

Avance de paulina llevado a plotly
